shell/platform/fuchsia/flutter/component_v2.cc

```diff
@@ -254,11 +254,11 @@ ComponentV2::ComponentV2(
   cloned_directory_ptr_.set_error_handler(
       [this](zx_status_t status) { cloned_directory_ptr_.Unbind(); });
 
-  // TODO(fxb/50694): Close handles from ComponentStartInfo::numbered_handles
+  // TODO(akbiggs): Close handles from ComponentStartInfo::numbered_handles
```

Comment:
Create a bug for the 3 TODO(akbiggs)?  Even something like "flutter_runner CFv2 cleanup"

Reply:
Done.